src/vtok_agent/src/agent/mod.rs

```diff
@@ -162,15 +190,16 @@ impl PostSyncAction {
             options.sync_interval_secs
         );
         match self {
-            Self::RestartNginx => Self::restart_webserver("nginx.service", options.reload_wait_ms),
-            Self::RestartHttpd => Self::restart_webserver("httpd.service", options.reload_wait_ms),
+            Self::RestartNginx => Self::restart_webserver("nginx", options.reload_wait_ms, None),
+            Self::RestartHttpd => Self::restart_webserver("httpd", options.reload_wait_ms, Some(defs::HTTPD_OVERRIDE_DATA)),
         }
     }
 
-    fn restart_webserver(service_name: &str, wait_ms: u64) {
+    fn restart_webserver(service: &str, wait_ms: u64, service_config_override: Option<&str>) {
+        let service_name = format!("{}.service", service);
```

Comment:
nit: .service string can be created directly inside of the function with config override